http/base/src/main/java/org/wildfly/security/http/HttpConstants.java

```diff
@@ -153,6 +153,7 @@ private HttpConstants() {
     public static final String FORM_NAME = "FORM";
     public static final String SPNEGO_NAME = "SPNEGO";
     public static final String BEARER_TOKEN = "BEARER_TOKEN";
+    public static final String OIDC_NAME = "OIDC";
```

Comment:
I wonder if at some point these should start to be deprecated - it feels odd we have mechanism specific constants centrally where we have split the mechanisms into their own modules.

Reply:
Good point. I can move `OIDC_NAME` to the `Oidc.java` class instead.